community/DOCS-CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+propose changes to this document in a pull request.
+
+## Before you begin
+
+### Code of conduct
+
+Knative follows the [Knative Code of Conduct](./CODE-OF-CONDUCT.md). By
+participating, you are expected to uphold this code. Please report unacceptable
+behavior to knative-code-of-conduct@googlegroups.com.
+
+### Style guide
+
+Knative documentation follows the
+[Google Developer Documentation Style Guide](https://developers.google.com/style/);
+use it as a reference for writing style questions.
+
+## Reporting documentation issues
+
+Knative uses Github issues to track documentation issues and requests. If you
+see a problem with the documentation, submit an issue using the following steps:
+
+1. Check the [Knative docs issues list](https://github.com/knative/docs/issues)
+   before creating an issue to avoid making a duplicate.
+2. Use the
+   [correct template](https://github.com/knative/docs/issues/new) for your new
+   issue. There are two templates available:
+    * **Bug report**:
+   If you're reporting an error in the existing documentation, use this
+   template. This could be anything from broken samples to typos. When you
+   create a bug report, include as many details as possible and include
+   suggested fixes to the issue. If you know which Knative component your
+   bug is related to, you can assign the appropriate
+   [Working Group Lead](./WORKING-GROUPS.md).
+    * **Feature request**:
+   For upcoming changes to the documentation or requests for more
+   information on a particular subject.
+
+Note that code issues should be filed against the
+[individual Knative repositories](http://github.com/knative),
+while documentation issues should go in the `docs` repository.
+
+## Contributing to the documentation
+
+### Working group
+
+The [Knative Documentation Working Group](./WORKING-GROUPS.md#documentation) meets weekly on
+Tuesdays and alternates between a 9am PT and a 4:30pm PT time to accomdate
+contributors in both the EMEA and APAC timezones. If you're interested in
+becoming more involved in Knative's documentation, start attending the working
+group meetings. You'll meet the writers currently steering our documentation
+efforts, who are happy to help you get started.
+
+### Getting started
+
+There are a couple different ways to jump in to the Knative doc set:
+
+* Look at issues in the Docs repo labled [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue).
+* Run through the [install guide](../install/README.md) for the platform of your
+  choice, as well as the [Getting Started with Knative App Deployment](../install/getting-started-knative-app.md) guide, and keep a [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging) of the experience.
+  What was hard for you? Then open a PR with a few improvements you could make
+  from the friciton log you kept. How can you make the documentation better so
```

Comment:
```suggestion
  from the friction log you kept. How can you make the documentation better so
```

Reply:
Also suggest rephrasing this paragraph, which feels a little disjointed. Maybe try putting both questions at the beginning and rewording the second sentence? 

 What was hard for you? How can you make the documentation better so that others don't run into the same issues you did? Open a PR with improvements based on your friction log.